### PR TITLE
Entity depiction and description

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: plugins/europeana/entity.js [hunk 1]
```diff
@@ -182,3 +182,39 @@ function getRelatedEntityTitleLink(entities) {
...
+ * If type is person, use biographicalInformation
+ * @return {String} a description when available in English
+ */
+export function getEntityDescription(type, entity) {
+  let description;
+
+  if (type === 'topic') {
+    description = entity.note && entity.note.en ? entity.note.en[0] : '';
+  } else if (type === 'person') {
+    if (entity.biographicalInformation.length !== undefined) {
```
Comment (rwd) [inline plugins/europeana/entity.js]: Can cause "Cannot read property 'length' of undefined" error, e.g. at /entity/person/18037-todd-woods

File: plugins/europeana/entity.js [hunk 2]
```diff
@@ -182,3 +182,39 @@ function getRelatedEntityTitleLink(entities) {
   }
   return entityDetails;
 }
+
+/** Get the description for the entity
```
Comment (rwd) [inline plugins/europeana/entity.js]: Put the description of the function on the following line, i.e. `/**` should be on its own line.

File: plugins/europeana/entity.js [hunk 3]
```diff
@@ -182,3 +182,39 @@ function getRelatedEntityTitleLink(entities) {
...
+ * If type is topic, use note
+ * If type is person, use biographicalInformation
+ * @return {String} a description when available in English
```
Comment (rwd) [inline plugins/europeana/entity.js]: Document the params.

File: tests/unit/plugins/europeana/entity.spec.js [hunk 4]
```diff
@@ -59,6 +72,13 @@ describe('plugins/europeana/entity', () => {
         const apiResponse = {
```
Comment (lbiedinger) [inline tests/unit/plugins/europeana/entity.spec.js]: just use ```const apiResponse = entitiesResponse.items[0];```

File: plugins/europeana/entity.js (hunk 3 above)
Comment (rwd) [reply plugins/europeana/entity.js]: Should be with @param, not @type. I have fixed.

**Pull request merged**